README.md

```diff
@@ -1,60 +1,98 @@
-[title]: - "AutoDock Vina"
+[title]: - "Running a Molecule Docking Job with AutoDock Vina"
 [TOC]
  
 ## Overview
-[AutoDock Vina](http://vina.scripps.edu/) is a molecular docking program useful for computer aided drug design.  In this tutorial, we will learn how to run AutoDock Vina on OSG.  We have a separate tutorial for screening a library of several ligands.
 
-## Tutorial files 
+[AutoDock Vina](http://vina.scripps.edu/) is a molecular docking program useful for computer aided drug design.  In this tutorial, we will learn how to run AutoDock Vina on OSG.  
+
+## Tutorial Files 
 
 It is easiest to start with the `tutorial` command. Type:
 
-	$ tutorial AutodockVina ### Creates "tutorial-AutodockVina" directory with input, job submission and execution script files
+	$ tutorial AutoDockVina
+
+This will create a directory `tutorial-AutodockVina`. Change into the directory and look at the available files: 
+
+	$ cd tutorial-AutoDockVina
+	$ ls
 
-This will create a directory `tutorial-AutodockVina`. Inside the directory, you will see the following files
+You should see the following: 
 
 	receptor_config.txt   # Configuration file (input)
-	receptor.pdbqt        # Receptor coordinates and atomic charges (input)
-	ligand.pdbqt          # Ligand coordinates and atomic charges (input)
+	data/
+		receptor.pdbqt    # Receptor coordinates and atomic charges (input)
+		ligand.pdbqt      # Ligand coordinates and atomic charges (input)
 	vina_job.submit       # Job submission file
 	vina_wrapper.bash     # Execution script
 
-We discuss the details of  `vina_job.submit` and `vian_wrapper.bash` files. To see how to prepare the input files, `receptor_config.txt`,  `receptor.pdbqt` and `ligand.pdbqt`, check the AutoDock website.
+We need to download the AutoDock program separately into the this directory as well. Go 
+to the [AutoDock Vina website](http://vina.scripps.edu/) and click on the Download link at the top of the page. Download the Linux version of the program; you can do this directly to the current directory by using the `wget` command and the download link: 
+
+	$ wget http://vina.scripps.edu/download/autodock_vina_1_1_2_linux_x86.tgz
```

Comment:
If you use -O for wget, you can give it a static name so the user doesn't have to replace the filename in the submit and wrapper scripts.

Not sure if this is desirable or not, but it would simplify things and prevent careless user errors.

Reply:
That's a good point! I like making versions obvious, in case it matters, but that would definitely help in terms of scripts. This wouldn't change the name of the folder inside though, right?

Reply:
No, the extracted folder would still need to be changed, unless there's some fancy tar magic I don't know off hand.

Reply:
I guess I could make a directory + tar into it...